src/Client/Resources/keen-io-3_0.php

```diff
@@ -65,7 +115,7 @@
         ),
 
         'getProject' => array(
-            'uri'         => 'projects/{projectId}',
+            'uri'         => 'organizations/{organizationId}/projects/{projectId}',
             'description' => 'GET returns detailed information about the specific project, '
```

Comment:
Maybe update this to `'Returns information about a project.`' to match https://keen.io/docs/api/#get-project.